### PR TITLE
Attempt to create bucket when creating s3 backend

### PR DESCRIPTION
**Pull request by pontus**

### CODE REVIEW

File: internal/storage/storage.go [hunk 1]
```diff
@@ -125,6 +126,22 @@ func newS3Backend(c S3Conf) *s3Backend {
...
+	_, err := s3.New(session).CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(c.Bucket),
+	})
+
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+
+			if aerr.Code() != s3.ErrCodeBucketAlreadyOwnedByYou &&
+				aerr.Code() != s3.ErrCodeBucketAlreadyExists {
+				log.Warning("Unexpected issue while creating bucket", err)
```
Comment (dtitov) [inline internal/storage/storage.go:140]: Maybe log.error?

File: internal/storage/storage_test.go [hunk 2]
```diff
@@ -208,8 +208,9 @@ func setupFakeS3() (err error) {
 	if err != nil {
 		if aerr, ok := err.(awserr.Error); ok {
 
-			if aerr.Code() != s3.ErrCodeBucketAlreadyOwnedByYou {
-				log.Warning("Unexpected issue while creating bucket", err)
+			if aerr.Code() != s3.ErrCodeBucketAlreadyOwnedByYou &&
+				aerr.Code() != s3.ErrCodeBucketAlreadyExists {
+				log.Warning("Unexpected issue while creating bucket: ", err)
```
Comment (dtitov) [inline internal/storage/storage_test.go:213]: Same here.

Comment (pontus) [reply internal/storage/storage_test.go]: Thanks, fixed.

File: internal/storage/storage.go (hunk 1 above)
Comment (pontus) [reply internal/storage/storage.go]: Thanks, fixed.

**Review by dtitov — APPROVED** [review]

**Pull request merged**